infra/README.md

```diff
@@ -1,8 +1,25 @@
-## Fetching Debian Bullseye Image and Verifying It
 
-- `./base-image/fetch-debian-image.sh`
+## Environment choice
+This section covers how to set a production-like environment. 
+For most developments, it is better to use the containers instead (see the dev-env repository). The infrastructure environment provides additional services like reverse-proxy (nginx) and monitoring, but is also much less practical.
 
-## Installing in a VM for Infrastructure Code Development
+There are 2 options, renting a server to a cloud provider, or creating a test VM for development:
+- Renting a server in the cloud is a simpler and faster way to get started. And closer to the production environment. Most cloud providers (Vultr, Digital Ocean, AWS...) offer free credit for new users, but it can get costly if you rent a powerful server and forget to stop it after use.
```

Comment:
je n'ai pas précisé mais vu que c'est du HTTP, tout est en clair et ça peut peut-être poser des problèmes de confidentialité

Reply:
with a public IP you can have HTTPS with Let's Encrypt certificates, it only requires setting an email address in the inventory (as specified in a comment)

Reply:
> Renting a server in the cloud is a simpler and faster way to get started
it's really subjective, I personally think it's the contrary